style.css

```diff
@@ -0,0 +1,1547 @@
+    font-size: 32px;
+    line-height: 44px;
+    text-align: center;
+    color: #172b4d;
+    margin-top: 69px;
+    margin-left: 23px;
+    margin-right: 25px;
+    position: absolute;
+  }
+
+  #contact-form {
+    display: flex;
+    width: 80%;
+    flex-direction: column;
+  }
+
+  .form {
+    position: absolute;
+    margin-top: 380px;
+    left: 50px;
+    width: 80%;
+    max-width: fit-content;
+  }
+
+  input {
+    width: 100%;
+    margin: 20px 0;
+    border: none;
+    border-bottom: 1px solid #dbd8d7;
+    height: 30px;
+  }
+
+  input::placeholder {
+    font-family: "Inter", serif;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    padding-left: 12px;
+  }
+
+  textarea {
+    width: 100%;
+    height: 114px;
+    margin-top: 15px;
+    border: none;
+    background-color: #fbf8f7;
+    background-image: url("icons/text-area-icon.png");
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    resize: none;
+  }
+
+  textarea::placeholder {
+    font-family: "Inter", serif;
+    font-size: 15px;
+    font-style: italic;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    height: 90px;
+    padding-left: 12px;
+    padding-top: 12px;
+    color: #3c3a39;
+  }
+
+  .cont-submit {
+    padding: 12px;
+    width: 100%;
+    max-width: fit-content;
+    height: 48px;
+    background: #ff6b00;
+    border: none;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 24px;
+    color: white;
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    border-radius: 3px;
+    font-size: 17px;
+    line-height: 24px;
+    justify-content: center;
+    display: flex;
+  }
+
+  .cont-submit:hover {
+    background-color: white;
+    color: #ff6b00;
+    border: 1px solid #ff6b00;
+  }
+
+  .cont-submit:visited {
+    background-color: #ff6b00;
+    color: white;
+    border: 1px solid #ff6b00;
+  }
+
+  .footer {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    justify-content: space-around;
+    margin-left: 98px;
+    margin-right: 98px;
+    margin-top: 25px;
+  }
+
+  .rect-footer {
+    margin-top: 797px;
+    width: 80%;
+    margin-left: auto;
+    margin-right: auto;
+    border: 1px solid #dfe1e6;
+  }
+
+  .buttons-up {
+    display: none;
+  }
+
+  .desktop-vector {
+    display: none;
+  }
+
+  .line {
+    display: none;
+  }
+
+  .text-line {
+    display: none;
+  }
+
+  .image2 {
+    display: none;
+  }
+}
+
+.container2 {
+  display: none;
+}
+
+.container1 {
+  width: 100%;
+}
+
+.form2 {
+  display: none;
+}
+
+.line3 {
+  display: none;
+}
+
+.grid-headline {
+  display: none;
+}
+
+@media screen and (min-width: 768.1px) {
+  body {
+    height: 4489px;
+    width: 100%;
+    background-color: white;
+  }
+
+  .up-side {
+    background-color: #3e3c3c;
+    width: 100%;
+    height: 72px;
+    position: absolute;
+
+    /* max-width: fit-content; */
+  }
```

Comment:
- [ ] For clean code and best practices reasons, kindly make sure you remove all commented and unused code lines from your code base